spec/classes/agent_spec.rb

```diff
@@ -18,7 +18,7 @@
     context "on #{os}" do
       systemd_fact = case facts[:osfamily]
                      when 'Archlinux', 'Fedora', 'Gentoo'
-                       { systemd: true }
+                       { systemd: true, service_provider: 'systemd' }
```

Comment:
we should really rework this. most other distributions also use systemd now.

Reply:
What do you like to do ?

`service_provider` fact is not work on the spec I don't know why, and that fail here https://github.com/puppetlabs/puppetlabs-postgresql/blob/main/manifests/params.pp#L85

Reply:
https://github.com/voxpupuli/puppet-zabbix/pull/834

Reply:
Thank, I've remove my patch for systemd and wait #834 for a rebase :)